internal/resources/metal/organization_member/resource.go

```diff
@@ -0,0 +1,289 @@
+package organizationmember
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path"
+	"strings"
+
+	equinix_errors "github.com/equinix/terraform-provider-equinix/internal/errors"
+	"github.com/equinix/terraform-provider-equinix/internal/framework"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/packethost/packngo"
+)
+
+type member struct {
+	*packngo.Member
+	*packngo.Invitation
+}
+
+func (m *member) isMember() bool {
+	return m.Member != nil
+}
+
+func (m *member) isInvitation() bool {
+	return m.Invitation != nil
+}
+
+func NewResource() resource.Resource {
+	return &Resource{
+		BaseResource: framework.NewBaseResource(
+			framework.BaseResourceConfig{
+				Name:   "equinix_metal_organization_member",
+				Schema: GetResourceSchema(),
+			},
+		),
+	}
+}
+
+type Resource struct {
+	framework.BaseResource
+}
+
+func (r *Resource) ImportState(
+	ctx context.Context,
+	req resource.ImportStateRequest,
+	resp *resource.ImportStateResponse,
+) {
+
+	var plan ResourceModel
+
+	parts := strings.Split(plan.ID.String(), ":")
+	invitee := parts[0]
+	orgID := parts[1]
+	idValue := fmt.Sprintf("%s:%s", invitee, orgID)
+
+	resp.Diagnostics.Append(resp.State.Set(ctx, idValue)...)
```

Comment:
`State.Set` populates the entire state. You need to use `SetAttribute` as in the `ImportState` defined in resource_base.go. Also, you need to get the id from the ImportStateRequest:

```suggestion
	var invitee, orgID any

	parts := strings.Split(req.ID, ":")

	if len(parts) != 2 {
		resp.Diagnostics.AddError(
			"Resource Import Passthrough Wrong ID Format",
			fmt.Sprintf("Expected import identifier with format: {invitee}:{organization_id}. Got: %q", req.ID),
		)
		return
	}
	invitee = parts[0]
	orgID = parts[1]

	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, fwpath.Root("invitee"), invitee)...)
	resp.Diagnostics.Append(resp.State.SetAttribute(ctx,  fwpath.Root("organization_id"), orgID)...)
```